UPGRADE-14.0.md

```diff
@@ -768,6 +768,29 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
     -   see #project-base-diff to update your project
 -   add Category ID to CategoryHierarchyItem ([#2962](https://github.com/shopsys/shopsys/pull/2962))
     -   see #project-base-diff to update your project
+-   product recalculations priority queue ([#2981](https://github.com/shopsys/shopsys/pull/2981))
+    -   `Shopsys\FrameworkBundle\Controller\Admin\ProductController` class was changed:
+        -   `editAction()` is now strictly typed
+        -   `newAction()` is now strictly typed
+    -   `Shopsys\FrameworkBundle\Model\Product\ProductFacade` class was changed:
+        -   `create()` method changed its interface:
+        ```diff
+        -   public function create(ProductData $productData)
+        +   public function create(
+        +       ProductData $productData,
+        +       ?ProductRecalculationPriorityEnumInterface $priority = null,
+        +   ): Product
```

Comment:
even though the real source code was changed this way, I would prefer to write the upgrade note like 

```
    public function create(
        ProductData $productData,
+       ?ProductRecalculationPriorityEnumInterface $priority = null,
-   )
+   ): Product
```

it seems more clear what was actually changed

Reply:
makes sense, that is really better - ef021407d